### PR TITLE
[ATH-1115] - Login

### PR DESCRIPTION
**Pull request by meganduffy**
The tightening of styles and full unit testing is pending.

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
+end
```
Comment (jamesfairbairn-wpengine) [inline app/helpers/sessions_helper.rb]: Unless you think you'll need this empty file, you can probably get rid of it.

**Comment by jamesfairbairn-wpengine** [comment]
Don't really have any major comments. When is the plan to add testing?

**Comment by meganduffy** [comment]
> Don't really have any major comments. When is the plan to add testing?

It looks as though the the story [`ATH-1117`](https://wpengine.atlassian.net/browse/ATH-1117) crosses over with this one substantially, I feel as though testing the system will be appropriate there.

**Comment by jamesfairbairn-wpengine** [comment]
> > Don't really have any major comments. When is the plan to add testing?
> 
> It looks as though the the story [`ATH-1117`](https://wpengine.atlassian.net/browse/ATH-1117) crosses over with this one substantially, I feel as though testing the system will be appropriate there.

Awesome! Just checking you had a plan for testing. Will approve this now then.

**Pull request merged**